x-pack/plugin/security/src/main/java/org/elasticsearch/xpack/security/authc/ApiKeyService.java

```diff
@@ -1309,12 +1311,12 @@ public void crossClusterApiKeyUsageStats(ActionListener<Map<String, Object>> lis
             listener.onResponse(Map.of());
             return;
         }
-        final SecurityIndexManager frozenSecurityIndex = securityIndex.freeze();
+        final SecurityIndexManager frozenSecurityIndex = securityIndex.defensiveCopy();
```

Comment:
I renamed this method `defensiveCopy` to avoid confusion with freezing and index which is completely unrelated to what this is doing (I didn't bother renaming the variables).